keyvault/client/secrets/src/main/java/com/azure/keyvault/SecretAsyncClient.java

```diff
@@ -0,0 +1,473 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.keyvault;
+
+import com.azure.core.ServiceClient;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.rest.PagedResponse;
+import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.SimpleResponse;
+import com.azure.core.http.rest.VoidResponse;
+import com.azure.core.implementation.RestProxy;
+import com.azure.keyvault.implementation.SecretBasePage;
+import com.azure.keyvault.models.DeletedSecret;
+import com.azure.keyvault.models.Secret;
+import com.azure.keyvault.models.SecretBase;
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.net.URL;
+import java.util.Objects;
+import java.util.function.Function;
+import com.azure.core.exception.ResourceNotFoundException;
+import com.azure.core.exception.ResourceModifiedException;
+import com.azure.core.exception.HttpRequestException;
+import com.azure.core.exception.ServerException;
+
+
+/**
+ * The SecretAsyncClient provides asynchronous methods to manage {@link Secret secrets} in the Azure Key Vault. The client
+ * supports creating, retrieving, updating, deleting, purging, backing up, restoring and listing the {@link Secret secrets}. The client
+ * also supports listing {@link DeletedSecret deleted secrets} for a soft-delete enabled Azure Key Vault.
+ *
+ * <p><strong>Samples to construct the client</strong></p>
+ * <pre>
+ * SecretAsyncClient.builder()
+ *   .endpoint("https://myvault.vault.azure.net/")
+ *   .credentials(keyVaultCredentials)
+ *  .build()
```

Comment:
These three lines of code are not aligned (`.build` is one space less)

Reply:
fixed